src/quick.py

```diff
@@ -0,0 +1,45 @@
+"""Quick sort."""
+
+
+def quick_sort(seq):
+    """Quicksort method."""
+    if isinstance(seq, list):
+        if len(seq) <= 1:
+            return seq
+        piv, seq = seq[0], seq[1:]
+        low, high = [x for x in seq if x <= piv], [x for x in seq if x > piv]
```

Comment:
While the double list comprehensions are fancy, you're iterating through the same list twice just to check two different conditions when you could iterate through the list once and check two conditions as you iterate